x-pack/plugins/observability_solution/dataset_quality/public/hooks/use_redirect_link.ts

```diff
@@ -18,20 +18,20 @@ import { LocatorPublic } from '@kbn/share-plugin/common';
 import { LocatorClient } from '@kbn/shared-ux-prompt-no-data-views-types';
 import { useKibanaContextForPlugin } from '../utils';
 import { BasicDataStream, TimeRangeConfig } from '../../common/types';
-import { useRedirectLinkTelemetry } from './use_telemetry';
+import { SendTelemetryFn } from './use_redirect_link_telemetry';
 
 export const useRedirectLink = <T extends BasicDataStream>({
   dataStreamStat,
   query,
   timeRangeConfig,
   breakdownField,
-  telemetry,
+  sendTelemetry,
 }: {
   dataStreamStat: T;
   query?: Query | AggregateQuery;
   timeRangeConfig: TimeRangeConfig;
   breakdownField?: string;
-  telemetry?: Parameters<typeof useRedirectLinkTelemetry>[0]['telemetry'];
+  sendTelemetry: SendTelemetryFn;
```

Comment:
Are we using this in this hook?

Reply:
yes - https://github.com/achyutjhunjhunwala/kibana/blob/80dabcf108a35f59137acd3125acc92a8e6e1af4/x-pack/plugins/observability_solution/dataset_quality/public/hooks/use_redirect_link.ts#L68-L78